common/laws/00_policing.txt

```diff
@@ -209,6 +190,71 @@ law_militarized_police = {
 			}
 		}
 	}
+	ai_enact_weight_modifier = {
+		#Petitions
+		value = 0
+		if = {
+			limit = {
+				has_journal_entry = je_government_petition
+				has_variable = desired_law_var
+				scope:law = var:desired_law_var
+			}
+			add = 750
+		}
+	}
+}
+
+law_disarmed_police = {
+	group = lawgroup_policing
+	
+	icon = "gfx/interface/icons/law_icons/militarized_police.dds"
+	
+	progressiveness = 100
+
+	unlocking_technologies = {
+		human_rights
+	}
+	unlocking_laws = {
+		law_guaranteed_liberties
+	}
+	modifier = {
+		country_institution_police_max_investment_add = 2
+	}
+
+	institution = institution_police
+	institution_modifier = {
+		state_radicals_from_sol_change_mult = -0.03
+		state_radicals_from_discrimination_mult = -0.10
+		interest_group_ig_trade_unions_pol_str_mult = 0.025
+		interest_group_ig_intelligentsia_pol_str_mult = 0.025
+		state_turmoil_effects_mult = 0.1
+	}
```

Comment:
This is a highly specific use-case, the effective 5% (total 25%) turmoil effect reduction make this institution nonviable. It either should have higher state_radicals_from_sol_change_mult as these policies only ever work with significant social investment, or it you should just bump the state_turmoil to an effective 50% percent at max level.

Reply:
Why would it be considered nonviable? the intent is to make it less effective at reducing turmoil effects as the police would be less effective at putting down angry civilians.

That being said, boosting the radicals from SoL change would be good too. The main goal of this one is mostly to reduce the influence of the more conservative igs though, as PB will get +50% from dedicated police now.

Reply:
So to get back at this comment, nonviable because 25% just isn't enough to counteract significant turmoil. In addition I would expect an alternative way to prevent turmoil altogether or to counteract it in other ways. For example radicals reduction for reduction or more authority to get more policies out.

Reply:
But the current fix is fine.